include/ur_api.h

```diff
@@ -2345,12 +2383,12 @@ urUSMDeviceAlloc(
 ///     - ::UR_RESULT_ERROR_OUT_OF_RESOURCES
 UR_APIEXPORT ur_result_t UR_APICALL
 urUSMSharedAlloc(
-    ur_context_handle_t hContext, ///< [in] handle of the context object
-    ur_device_handle_t hDevice,   ///< [in] handle of the device object
-    ur_usm_desc_t *pUSMDesc,      ///< [in][optional] USM memory allocation descriptor
-    ur_usm_pool_handle_t pool,    ///< [in][optional] Pointer to a pool created using urUSMPoolCreate
-    size_t size,                  ///< [in] size in bytes of the USM memory object to be allocated
-    void **ppMem                  ///< [out] pointer to USM shared memory object
+    ur_context_handle_t hContext,  ///< [in] handle of the context object
+    ur_device_handle_t hDevice,    ///< [in] handle of the device object
+    const ur_usm_desc_t *pUSMDesc, ///< [in][optional] Pointer to USM memory allocation descriptor.
```

Comment:
@veselypeta : should we have here both, device and host descriptors?

Reply:
We decided to chain them instead of adding additional parameters to the function. See the example [here](https://github.com/oneapi-src/unified-runtime/blob/main/test/conformance/usm/urUSMSharedAlloc.cpp#L42)